bse/bseitem.cc

```diff
@@ -1335,4 +1337,16 @@ ItemImpl::apply_idl_property_need_undo (const StringVector &kvlist)
   return !Aida::aux_vector_check_options (kvlist, "", "hints", "skip-undo");
 }
 
+void
+ItemImpl::block_property_undo()
+{
```

Comment:
I really don't want to introduce these. We already have other mechanism that check if undo steps should be recorded at all, and we have a dummy undo stack `bse_undo_stack_dummy` that could be used if we'd want to temporarily ignore undo steps. Also, the *C++* developer shouldn't be bothered with having to use block/unblock in pairs, so for the future, similar API should make use of ctor/dtor pairs like std::lock_guard does.

Assuming your branch starts off from `master`, you can get rid of this commit with `git rebase -i` and then remove the `pick ...` line that refers to this commit.

Reply:
Ok, I just tried to do a 1:1 port so I had to do something where the old code blocked undo. If you say you believe it is not necessary, thats even better, because the code looks cleaner now. I removed block/unblock now.